tcms/core/tests/test_admin.py

```diff
@@ -116,14 +118,13 @@ def test_delete_another_user(self):
 
     def test_delete_myself(self):
         self.client.login(  # nosec:B106:hardcoded_password_funcarg
-            username=self.tester.username,
+            username=self.superuser.username,
             password='password')
         response = self.client.get(self.url)
 
         # verify there's the Yes, I'm certain button
         self.assertEqual(HTTPStatus.OK, response.status_code)
         self.assertContains(response, "Yes, I'm sure")
```

Comment:
this is probably translated in the response and that's why tests in CI with different locales fail. Try marking the string in `_()`